dev/io.openliberty.session.6.1.internal/src/io/openliberty/session61/http/impl/HttpSessionImpl61.java

```diff
@@ -0,0 +1,116 @@
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.session61.http.impl;
+
+import java.util.function.Consumer;
+import java.util.logging.Level;
+
+import com.ibm.ejs.ras.TraceNLS;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.ws.session.AbstractSessionData;
+import com.ibm.ws.session.SessionContext;
+import com.ibm.ws.session.utils.LoggingUtil;
+import com.ibm.wsspi.session.ISession;
+
+import jakarta.servlet.ServletContext;
+import jakarta.servlet.http.HttpSession;
+
+/**
+ * Create instance and pass the ISession, along with others contexts, into the parent's hierarchy
+ */
+public class HttpSessionImpl61 extends AbstractSessionData {
+    private static final String CLASS_NAME = "HttpSessionImpl61";
+    private static TraceNLS nls = TraceNLS.getTraceNLS(HttpSessionImpl61.class, "io.openliberty.session61.resources.SessionMessages");
+
+    protected HttpSessionImpl61(ISession session, SessionContext sessCtx, ServletContext servCtx) {
+        super(session, sessCtx, servCtx);
+
+        if (TraceComponent.isAnyTracingEnabled() && LoggingUtil.SESSION_LOGGER_CORE.isLoggable(Level.FINER)) {
+            LoggingUtil.SESSION_LOGGER_CORE.log(Level.FINER, CLASS_NAME + " Constructor ISession [" + session + "] ; SessionContext [" + sessCtx + "] , servletContext [" + servCtx
+                                                             + "] this [" + this + "]");
+        }
+    }
+
+    private static class SessionDataAccessor implements HttpSession.Accessor {
+        private final String INNER_CLASS_NAME = "SessionDataAccessor";
+        private final ISession iSession;
+        String savedSessionId = null;
+
+        public SessionDataAccessor(ISession session, String id) {
+            iSession = session;
+            savedSessionId = id;
+
+            if (TraceComponent.isAnyTracingEnabled() && LoggingUtil.SESSION_LOGGER_CORE.isLoggable(Level.FINER)) {
+                LoggingUtil.SESSION_LOGGER_CORE.log(Level.FINER,
+                                                    INNER_CLASS_NAME + " Constructor , session id [" + savedSessionId + "] , this [" + this + "] . ISession [" + iSession + "]");
+            }
+        }
+
+        /**
+         * Wrap and create a HttpSession reference which application can access via the passed in {@code sessionConsumer}
+         * Update the last accessed time.
+         */
+        @Override
+        public void access(Consumer<HttpSession> sessionConsumer) {
+            if (TraceComponent.isAnyTracingEnabled() && LoggingUtil.SESSION_LOGGER_CORE.isLoggable(Level.FINER)) {
+                LoggingUtil.SESSION_LOGGER_CORE.entering(CLASS_NAME, INNER_CLASS_NAME, "access");
```

Comment:
I suggest removing CLASS_NAME and only have INNER_CLASS_NAME.  The signature, that this ends up using thinks CLASS_NAME is the class, INNER_CLASS_NAME is the method name and "access" is a parameter.